### ISSUE TITLE
Bad font attributes reported in Firefox

### ISSUE THREAD

**Issue by spl** [box/viewer.js#164]
For the page below, Firefox (Mac OS X) reports various errors about the font CSS in the console.

https://app.doxiq.com/d/Fs6td/NATO-Allied-Joint-Doctrine-for-Psychological-Operations

The errors are of the form:

```
downloadable font: maxp: bad max_zones: 0 (font-family: "crocodoc-w6KLqh-inv-f6364" style:normal weight:normal stretch:normal src index:0)
source: data:font/ttf;base64,AAEAAAAK[...]
downloadable font: OS/2: bad weight: 1 (font-family: "crocodoc-w6KLqh-inv-f6364" style:normal weight:normal stretch:normal src index:0)
source: data:font/ttf;base64,AAEAAAAK[...]
```

These errors do not appear in Chrome or Safari on Mac OS X. We're using Viewer.js v0.8.0.

**Comment by lakenen** [comment]
This looks like an issue with conversion rather than viewer.js. I've opened a ticket internally. I'll update this issue when we figure out what's going on. 

Does this happen with any other documents? Does it affect the rendering at all? It seems like the fonts show up correctly, but I might be missing something.

**Comment by spl** [comment]
> Does this happen with any other documents?

This is the first time I've noticed the error messages. I only noticed the errors because I was looking into an issue and using this document as a test case.

> Does it affect the rendering at all? It seems like the fonts show up correctly, but I might be missing something.

I did a quick browse through the document to look for different fonts that might be rendered incorrectly. I found:

1. Page 37, Figure 3-2. Manning options for PSYOPS - The font in the diagram appears different in Box vs. Preview. This, however, is `crocodoc-w6KLqh-inv-f6366`, not `crocodoc-w6KLqh-inv-f6364` as noted above.

`crocodoc-w6KLqh-inv-f6364` is apparently the font used in the page headers (e.g. “NATO/PfP UNCLASSIFIED” and “AJP-3.10.1(A)”.

My conclusion: It doesn't noticeably affect rendering.
